utilities_common/sfp_helper.py

```diff
@@ -43,6 +43,295 @@
 
 CMIS_DATA_MAP = {**QSFP_DATA_MAP, **QSFP_CMIS_DELTA_DATA_MAP}
 
+
+QSFP_STATUS_MAP = {
```

Comment:
@longhuan-cisco is this for all transceivers or CMIS/CCMIS only?

Reply:
This is for all transceivers. since status API was added for non-CMIS optics via a another PR.

Reply:
a NON_CMIS_STATUS_MAP was added for non cmis transceivers.